packages/components/Button/src/Button.styling.ts

```diff
@@ -95,6 +98,37 @@ export const getDefaultSize = (): ButtonSize => {
   return 'medium';
 };
 
+export const getPlatformSpecificAppearance = (appearance: ButtonAppearance): ButtonAppearance => {
+  // 'Outline' exists only for Mobile platforms, maps to default on other platforms.
+  const supportsOutlineAppearance = Platform.OS === 'android' || Platform.OS === 'ios';
+
+  // Mobile platforms do not have seperate styling when no appearance is passed.
+  const hasDifferentDefaultAppearance = !(Platform.OS === 'android' || Platform.OS === 'ios');
+
+  switch (appearance) {
+    case 'accent': // Included to cover Mobile platform naming guidelines, maps to 'primary'.
+      return 'primary';
+
+    case 'outline':
+      if (supportsOutlineAppearance) {
+        return 'outline';
```

Comment:
Even if outline is passed in appearance w/o this code segment, it would still be rendered as default. Had added this line for uniformity in covering all the appearance options

Reply:
Replaced explicit handling to remove the notion of mapping. Code handling responsibility is on platform. Passing outline will still lead to default styles being picked due to lack of outline specific Tokens for unsupported platforms